### ISSUE TITLE
Documentation/discussion: Graph dependency hierarchy using ASCII art in case it develops branching? 

### ISSUE THREAD

**Issue by russelljjarvis** [scidash/docker-stacks#5]
Discussion point in the documentation for this image, should we graph dependency hierarchy using ASCII art in case it develops branching? 
Opinion wanted.

Perhaps this is easier on eyes/brain (please imagine that arrows are properly aligned)?

       jupyter/scipy-notebook
                     |
                     V
        scipy-notebook-plus
                     |
                     V
       neuron-mpi-neuroml
                     |
                     V
     neuronunit-scoop-deap
                     |
                     V
               sciunitopt

**Comment by russelljjarvis** [comment]
Solution use the prettier dependency graph.

**Comment by rgerkin** [comment]
@russelljjarvis 
I've used [this](http://graphviz.org/) (or actually the python front-end to it) to generated reasonably nice-looking graphs from a file expressing the structure of the graph.  If it gets much more complicated we can try that, but if the relationships are fairly linear and/or logical we can probably just use something like the graph I made, for which I used [this](draw.io).  If I move it to Google Drive I think it is possible to do real-time collaboration on it.
